### PR TITLE
WFLY-12586 Duplicated classes in test suite, content slightly differs in some cases

### PR DESCRIPTION
**Pull request by parsharma**
Issue: https://issues.redhat.com/browse/WFLY-12586

### CODE REVIEW

File: testsuite/shared/src/main/java/org/jboss/as/test/integration/common/WebInfLibClass.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package org.jboss.as.test.integration.common;
```
Comment (yersan) [inline testsuite/shared/src/main/java/org/jboss/as/test/integration/common/WebInfLibClass.java:1]: Would you mind moving also the license header to the common file? I would say it is not mandatory for merging, but we should keep the pattern we already have.

Comment (darranl) [inline testsuite/shared/src/main/java/org/jboss/as/test/integration/common/WebInfLibClass.java:1]: Actually if the header is missing this is mandatory for merging, I know some other projects have relaxed that requirement but the WildFly Core and WildFly repos have mixed licensing so I believe we do still require the header to correctly identify on a file by file basis.

**Review by darranl — CHANGES_REQUESTED** [review]
Missing header.

File: testsuite/shared/src/main/java/org/jboss/as/test/integration/common/WebInfLibClass.java (hunk 1 above)
Comment (parsharma) [reply testsuite/shared/src/main/java/org/jboss/as/test/integration/common/WebInfLibClass.java:23]: Sure. I will move the license header.

**Review by darranl — APPROVED** [review]

**Pull request merged**